app/services/record_participant_declaration.rb

```diff
@@ -1,110 +1,28 @@
 # frozen_string_literal: true
 
-require "json_schema/validate_body_against_schema"
-
 class RecordParticipantDeclaration
```

Comment:
What job does this class do now? As I see it, not much of one. Maybe we should cut out the middle man and go call our factory in `ParticipantDeclarationsController`?

Reply:
I think this class now does even less of a job than it used to when I wrote this comment.

I'm like 90% sure we should inline it - but we can probably do it in a later PR.